conf/mendel.config

```diff
@@ -2,21 +2,23 @@
 params {
   config_profile_name = 'MENDEL'
   config_profile_description = 'GMI MENDEL cluster profile provided by nf-core/configs'
-  config_profile_contact = 'Philipp H (@phue)'
+  config_profile_contact = 'Patrick Hüther (@phue)'
```

Comment:
I guess that was my fault  - sorry about that!

Reply:
No worries :)